css_01/css_01.css

```diff
@@ -0,0 +1,49 @@
+#bodyContent
+{
+	margin: auto;
+	width: 380px;
+}
+
+#items
+{
+	border: solid;
+	border-width: 1.5px;
+	border-color: gray;
+	background: #E8E8E8;
+	padding-top: 20px;
+	padding-left: 20px;
+	padding-bottom: 15px;
+	padding-right: 24px;
+}
+
+#title
+{
+	background: #66FF00;
+	text-align: center;
+	font-family: Sans-serif;
+	font-weight: bold;
+	color: white;
+	padding: 8px;
+}
+
+#email, #firstName, #lastName, #Password, #Confirm_Password
```

Comment:
Here you could add a single class to all of the html elements instead of adding style to each one of them separately.